regression/cbmc/unknown-argument-suggestion/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+dummy.c
+--traec
+did you mean --trace
+^EXIT=[123456789]\d*$
```

Comment:
A right nuisance: Apparently the error codes on windows and linux are different, but our test.pl script insist on very particular exit code checks. I'm tempted to just remove this test to be honest.